template/src/main/java/com/asyncapi/Config.java

```diff
@@ -0,0 +1,19 @@
+package {{ params['userJavaPackage'] }};
```

Comment:
The library defines a bunch of components and beans.
For Spring to find and automatically load all these beans on startup, I added the @ComponentScan to the Config class.

Now, @ComponentScan searches the package of its class and all subpackages.
I therefor put the Config class into the top level package, so that all packages are searched and all beans and components are found...